packages/web-app-preview/src/App.vue

```diff
@@ -267,7 +267,7 @@ export default defineComponent({
     ...mapGetters(['capabilities', 'user']),
 
     pageTitle() {
-      const translated = this.$gettext('Preview for %{currentMediumName}')
+      const translated = this.$gettext('Preview for %{currentMediumName}') || ''
```

Comment:
Don't think this makes sense either

Reply:
Why not? IIRC the `<h1>` (="page title", even though there's a title in the HTML's head also) with a translated hint for e.g. screenreaders on what's displayed on the screen is necessary from an a11y standpoint

Reply:
Or are you talking about the right part of the OR? ;)

Reply:
Yes, we had issues when we passed undefined into `$gettext()` but the `|| ''` doesn't help with that.